### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OlivierBondu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```
Comment (swertz) [inline README.md:17]: Why remove this? plotIt works outside of CMS as well... But it's true that the `setup_sl6_env.sh` file is not up-to-date anyway.

Comment (OlivierBondu) [reply README.md:17]: sure, I'd just prefer the 'default' install to be within CMSSW... I just remove it from the readme, I did not touch the script ?

Equally I could mention this script in a comment in the README itself, I don't mind

Comment (blinkseb) [reply README.md:17]: If you remove that, you have to source `setup_for_cms_env.sh` otherwise it won't work

Comment (blinkseb) [reply README.md:17]: Just to be clear:

 - `setup_sl6_env.sh` just load a more recent version of gcc and boost, using `cvmfs`. It's basically equivalement of doing `module load gcc/...` in ingrid, but it also works on lxplus.
- `setup_for_cms_env.sh` setup the env. for compiling plotIt within a CMSSW env. All it does is exporting `BOOST_ROOT` in order for the Makefile to use the correct boost version (the one from CMSSW, not the system one)

Comment (OlivierBondu) [reply README.md:17]: I fully agree with sourcing the other, see the full diff :wink: 

my current worry is that the default is compiling without cmsenv, so default execution should be without cmsenv as well... which is a bit counter intuitive as I think 'default' install for most of us is within CMSSW anyway

Comment (blinkseb) [reply README.md:17]: ... sorry, missed the full diff ...

**Comment by OlivierBondu** [comment]
Updated, hopefully it's more clear this way ?

**Comment by swertz** [comment]
Yep thanks!

**Pull request merged**